src/web/modules/transactions/fix_failed_stripe_refund.ts

```diff
@@ -0,0 +1,34 @@
+import {NextFunction, Request, Response} from "express";
+import {Connector, Ledger} from "../../../lib/pay-request/client";
+
+export async function show(req: Request, res: Response, next: NextFunction): Promise<void> {
+    try {
+        const transaction = await Ledger.transactions.retrieve(req.params.id)
+
+        res.render(`transactions/refundConfirmation`, {
+            transaction
+        })
+    } catch (error) {
+        next(error)
+    }
+}
+
+
+export async function fixFailedStripeRefund(req: Request, res: Response, next: NextFunction): Promise<void> {
+    try {
+        const transaction = await Ledger.transactions.retrieve(req.params.id)
+        const gatewayAccountId = transaction.gateway_account_id
+        const refundId = transaction.transaction_id
+        const chargeId = transaction.parent_transaction_id
+        const zendesk_ticket_id = req.body.zendesk_ticket_id
+        const github_user_id = req.user && req.user.username
+
+       await Connector.failedStripeRefund.fixStripeRefund(gatewayAccountId, refundId, chargeId, {zendesk_ticket_id, github_user_id}  )
```

Comment:
Just a stylistic thing but I’d pull the `{zendesk_ticket_id, github_user_id}` bit out to make it more prominent:

```javascript
await Connector.failedStripeRefund.fixStripeRefund(gatewayAccountId, refundId, chargeId,
    {
        zendesk_ticket_id,
        github_user_id
    }
)
```